client/src/components/CommentsList.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import CommentEntry from './CommentEntry';
 
 const CommentsList = props => (
-  <div className='comments'>
+  <div>
     {props.commentsList.map(comment => <CommentEntry key={comment.id} comment={comment} />
```

Comment:
I think the style guide will require you to use destructuring assignment for props
https://medium.freecodecamp.org/the-basics-of-destructuring-props-in-react-a196696f5477